lib/json_api_resource/handlers/find_handler.rb

```diff
@@ -0,0 +1,19 @@
+module JsonApiResource
+  module Handlers
+    class FindHandler
+      attr_accessor :result
+
+      def initialize(results)
+        simplify!(results)
+      end
+
+      def simplify!(results)
+        result = results.first
+
+        result.meta = results.meta
+
+        result.linked_data = results.try(:linked_data)
+      end
```

Comment:
No idea. I haven't looked at this in like 6 months. I think that was a joke. This probably doesn't run. I'm not sure why this method is necessary anyway i it's only called in the initializer